mscore/exportxml.cpp

```diff
@@ -5437,12 +5441,12 @@ bool saveXml(Score* score, const QString& name)
       QFile f(name);
       if (!f.open(QIODevice::WriteOnly))
             return false;
-      
+
```

Comment:
This appears to be github getting confused.  - on one side and a corresponding + on the other side, both for empty lines.

Reply:
Not really an empty line, but a removed whitespace (a tab probably, or several spaces)

Reply:
Probably done by Qt Creator, which I have set to strip trailing whitespace.